src/styles/variables.scss

```diff
@@ -0,0 +1,16 @@
+$calendar-bcg-color: #eee;
+$calendar-gap: 1px;
+$calendar-width: 100px;
+$calendar-height: 100px;
+$calendar-hover-bcg-color: #ffbfcb;
```

Comment:
The variables for colors should be named in a way that describes their purpose or their color, not where they are used. For example, instead of `$calendar-bcg-color`, consider `$color-light-gray` or `$color-background`, depending on its usage. This makes it easier to reuse variables and understand their purpose in different contexts.